Resources/Maps/_LostParadise/LPPKerberos.yml

```
@@ -18726,10 +18726,10 @@ entities:
           - 0
           - 0
         - volume: 2500
-          temperature: 234.99979
+          temperature: 293.15
           moles:
-          - 20.078888
-          - 75.53487
+          - 21.6852
```

yamllint

[error] wrong indentation: expected 12 but found 10

(indentation)